src/processTranslation.tsx

```diff
@@ -49,14 +49,11 @@ export const processTranslation = ({
     throw new Error(`Missing translation: "${key}"`);
   }
 
-  let suffix;
-
-  if (args.context) {
-    suffix = args.context;
+  let suffix = args.context;
 
   // @TODO: Update to the new format https://www.i18next.com/translation-function/plurals
-  } else if (typeof args[COUNT] !== 'undefined' && args[COUNT] !== 1 && args[COUNT] !== -1) {
-    suffix = 'plural';
+  if (typeof args[COUNT] !== 'undefined' && args[COUNT] !== 1 && args[COUNT] !== -1) {
+    suffix = suffix ? `${suffix}_plural` : 'plural';
```

Comment:
Is this breaking in that consumers need to check now that their context'd translations have a `_plural` translation present as well?

Reply:
No, it's not a breaking change, the logic is the same as before. 
Line 59 has a fallback to `root[lastPart]` when there is no `_plural`

Reply:
Thanks for clarifying